frontend/src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,16 @@
+import { Navbar, Nav, Button } from 'react-bootstrap';
+
+const NavBar = () => {
+  return(
+    <>
+  <Navbar bg="light" variant="light">
+    <Navbar.Brand href="#home"><b>BUSICARD</b></Navbar.Brand>
+    <Nav className="ml-auto">
+      <Nav.Link href="#home"><Button variant="outline-dark">Log Out</Button></Nav.Link>
+    </Nav>
+  </Navbar>
+</>
+  )
```

Comment:
Can we get this code formatted / following code style guides. Try merging in master and making a commit, the new webhook should automatically format pre commit.